transformation_models.py

```diff
@@ -886,7 +924,7 @@ def TTT(self, fs=1e-2, ff=.99, ax=None, **kwargs):
         ax.set_xscale('log')
         ax.set_xlabel('Time (s)')
         ax.set_ylabel(u'Temperature (°C)')
-        ax.set_title(self.alloy.format_composition())
+        ax.set_title(f'{self.alloy.format_composition()} - Convection Coefficient (h): {h} W/m²K')
```

Comment:
There is probably a smarter way of adding the convection coefficient for the title. This way if the person wants to plot only the TTT diagram having h in the title dosen't make sense.